### ISSUE TITLE
Server side verification API for Java

### ISSUE THREAD

**Issue by poudel** [khalti/khalti-docs-official-repo#6]
An example for server side verification for Java would be a great addition.

http://docs.khalti.com/api/verification/

**Comment by shirshak55** [comment]
I don't think it will add that much value. 

This one is self documented and I think any good java developer can understand it.


curl https://khalti.com/api/v2/payment/verify/ \
   -H "Authorization:Key test_secret_key_f59e8b7d18b4499ca40f68195a846e9b" \
   -d amount=1000 \
   -d token=QUao9cqFzxPgvWJNi9aKac

**Comment by poudel** [comment]
I agree that `any good java developer` might not need such an example. But what about the rest of the Java developers? :)

In any case, I think having an example would be good for consistency and didactic purposes.

**Comment by shirshak55** [comment]
@poudel yea but are u getting request from java developer regarding docs? Because u know most people don't use java for web but for android app only. Yea if there are more request then it is wise to make a good docs and examples :)
